packages/lit-html/src/test/directives/style-map_test.ts

```diff
@@ -42,6 +42,37 @@ suite('styleMap', () => {
     container = document.createElement('div');
   });
 
+  test('render() only properties', () => {
+    // Get the StyleMapDirective class indirectly, since it's not exported
+    const result = styleMap({});
+    const StyleMapDirective = result._$litDirective$;
```

Comment:
Maybe a TODO to revisit this?

Reply:
I think this is ok actually. It's either this or exporting the class.